handbook/openid-connect/openid-connect.md

```diff
@@ -258,35 +256,55 @@ If the provider is https with a certificate signed by a CA not included in the N
 Claims are returned as a JSON object.  
 
 !!! note ""
-    Example response from `userinfo_endpoint`. Any of these claims may be used to map a user to a role in a workspace.
+    Example response from `userinfo_endpoint`. Use any of these claims to a map a user to a role in a workspace.
+    ```json
+    {
+        "email": "mark.black@ni.com",
+        "family_name": "Black",
+        "given_name": "Mark",
+        "name": "Black",
+        "ni_employee": "2670",
+        "sub": "mblack"
+    }
+    ```
+
+    Within the security UI the claim and its returned value can be mapped to a role within a Workspace.
+
+    <figure>
+      <img src="../claim-mapping.png" width="500" />
+      <figcaption>Mapping the ni_employee claim to a workspace.</figcaption>
+    </figure>
+
+If the claim value is a scalar, then it must exactly match the value specified in the role mapping for the mapping to apply to the user. If the claim value is an array, then one of the array elements must exactly match the value specified in the role mappings.
 
-```json
-{
-    "email": "mark.black@ni.com",
-    "family_name": "Black",
-    "given_name": "Mark",
-    "name": "Black",
-    "ni_employee": "2670",
-    "sub": "mblack"
-}
-```
+If the claim value contains quotes the quotes must be escaped.
 
-Within the security UI the claim and its returned value can be mapped to a role within a Workspace.
+!!! note "Example claim containing quotes"
 
-<figure>
-  <img src="../claim-mapping.png" width="500" />
-  <figcaption>Mapping the ni_employee claim to a workspace.</figcaption>
-</figure>
+    ```json
+      "userinfo": {
```

Comment:
Is this missing an opening { or something to make it a complete JSON document?

Reply:
Fixed!